test/runtests.jl

```diff
@@ -8,12 +8,8 @@ function tempfitsfile(fn)
         fitsfile = fits_clobber_file(filename)
         fn(fitsfile)
 
-        # Write arbitrary data to file to avoid errors on closing it
         if fitsfile.ptr != C_NULL
-            temp = ones(1)
-            fits_create_img(fitsfile, temp)
-            fits_write_pix(fitsfile, temp)
-            close(fitsfile)
```

Comment:
Are you sure it's safe to remove the call to `close`?  Open handles have always caused problems for me on Windows

Reply:
From what I understand by [reading the docs](https://heasarc.gsfc.nasa.gov/docs/software/fitsio/c/c_user/node35.html), `fits_delete_file` will close and delete the file, ignoring errors if any. This gets around the fact that `fits_close_file` errors while closing an empty file. I'm not totally sure how this works on windows, but tests seem to pass.